cdap-ui/app/cdap/components/DataPrep/DataPrep.scss

```diff
@@ -31,8 +31,8 @@ $toggle-btn-disabled-bg-color: #cccccc;
     .dataprep-body.row {
       // height calculations
       // 100vh - height of navbar - top panel - footer
-      // 100vh - 50px - 50px - 54px
-      height: calc(100vh - 154px);
+      // 100vh - 50px - 548px - 54px
```

Comment:
This is fixed in the styling PR i have open in a more generic way. https://github.com/cdapio/cdap/pull/11020/files#diff-f986deff3a889570757d014a947fc2f9R37 Can you skip this change for now?

Reply:
Yeah, I will rebase this branch with your PR once yours is merged